hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -126,7 +126,7 @@ private static boolean checkAccessInAcl(OzoneAcl a, String[] groups,
   public static boolean checkAclRights(List<OzoneAcl> acls,
       RequestContext context) throws OMException {
     String[] userGroups = context.getClientUgi().getGroupNames();
-    String userName = context.getClientUgi().getUserName();
+    String userName = context.getClientUgi().getShortUserName();
```

Comment:
In the failed acceptance test, acls were added with the Full principle name (links.robot line 45-49) so changing the compare here with shortName will mismatch and fail.

Reply:
note for reproduce the test case 
```
cd compose/ozonesecure
docker-compose up -d
docker-compose --scale datanode=3
docker-compose exec scm bash
robot smoketest/basic/links.robot
```